lib/ufe/ProxyShapeHierarchy.h

```diff
@@ -68,9 +72,9 @@ class MAYAUSD_CORE_PUBLIC ProxyShapeHierarchy : public Ufe::Hierarchy
 	const UsdPrim& getUsdRootPrim() const;
 
 private:
-	Ufe::SceneItem::Ptr fItem;
-	Hierarchy::Ptr fMayaHierarchy;
-	Ufe::HierarchyHandler::Ptr fMayaHierarchyHandler;
+	Ufe::SceneItem::Ptr fItem{nullptr};
+	Hierarchy::Ptr fMayaHierarchy{nullptr};
+	Ufe::HierarchyHandler::Ptr fMayaHierarchyHandler{nullptr};
```

Comment:
Is the explicit `nullptr` initialization here needed for some reason? It looks like these are `std::shared_ptr` so shouldn't they be empty by default?

Reply:
Reverted in
https://github.com/Autodesk/maya-usd/pull/237/commits/f53f29c7258bc1d99897d843dbfef2e0316dd702